File: customer-salsel-ui/Salassel-Customer-Ui/src/app/Model/productFieldValuesDto.ts
export interface IProductFieldValuesDto {
    id?: number,
    name?: string,
    status?: string,
  }

  export class ProductFieldValuesDto implements IProductFieldValuesDto {
    constructor(
        public id?: number,
        public name?: string,
        public status?: string,
    ) {}
  }

File: salsel-backend/src/main/java/com/salsel/dto/ProductFieldValuesDto.java
package com.salsel.dto;

import com.salsel.model.ProductField;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@AllArgsConstructor
@NoArgsConstructor
@Data
public class ProductFieldValuesDto {
    private Long id;
    private String name;
    private String status;
}
